pyproject.toml

```diff
@@ -17,6 +17,7 @@ python = "^3.8"
 pytorch-lightning = "^1.0.8"
 scipy = "^1.4.1"
 torch = "1.7.1"
+sphinx-rtd-theme = "^0.5.2"
```

Comment:
I'd think of this as a development dependency

Reply:
That's right - fixed that